src/Meta/MetadataInterface.php

```diff
@@ -13,41 +13,80 @@
 
 namespace Sonata\BlockBundle\Meta;
 
-/**
- * @author Hugo Briand <briand@ekino.com>
- *
- * NEXT_MAJOR: Remove CoreBundle dependency
```

Comment:
This should be replaced by the BlockBundle MetadataInterface , or that is what the CoreBundle says